packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -1,57 +1,44 @@
-import { AzureKeyVaultClient } from '@celo/contractkit/lib/utils/azure-key-vault-client'
-import threshold from 'blind-threshold-bls'
-import { ErrorMessages } from '../common/error-utils'
+import threshold_bls from 'blind-threshold-bls'
 import logger from '../common/logger'
-import config, { DEV_MODE, DEV_PRIVATE_KEY } from '../config'
+import config from '../config'
 
+export interface ServicePartialSignature {
+  url: string
+  signature: string
+}
+
+function flattenSigsArray(sigs: Uint8Array[]) {
+  return Uint8Array.from(sigs.reduce((a, b) => Array.from(a).concat(Array.from(b)), [] as any))
```

Comment:
Is the `Array.from(a)` necessary given that it's initialized to be an array? I'm thinking something like:
`sigs.reduce((a, b) => a.concat(Array.from(b)), [] as any)`

Reply:
I can try it -- I copied from this example
https://github.com/celo-org/blind-threshold-bls-wasm/commit/e1e2f8a1ab5154c2f0b1c55cb0d61fbb1d907208